understanding/20/abbreviations.html

```diff
@@ -11,11 +11,12 @@ <h1>Understanding Abbreviations</h1>
    <section id="brief">
       <h2>In brief</h2>
       <dl>
-         <dt>Objective</dt><dd>Users can identify and learn what abbreviations mean</dd>
-         <dt>Author task</dt><dd>Provide the expanded form of abbreviations to users</dd>
+         <dt>Goal</dt><dd>Users can identify and learn what abbreviations mean.</dd>
+         <dt>What to do</dt><dd>Provide the expanded form of abbreviations to users.</dd>
+         <dt>Why it's important</dt><dd>More people, especially those with cognitive disabilities, can understand the meaning of content.</dd> 
```

Comment:
```suggestion
         <dt>Why it's important</dt><dd>Some people, especially those with cognitive disabilities, may not understand the shortened form.</dd> 
```

Reply:
better!

Reply:
Yeah, the reason I went with this is that there was prior discussion that we don't just want to repeat the same term all the time -- this way someone who doesn't entirely understand what an abbreviation is gets an idea it is a shortened form (which can be expanded).

Reply:
I'm inclined to even make it a bit more wordy to add clarity, drawing from the definition:
```suggestion
         <dt>Why it's important</dt><dd>Some people, including those with cognitive disabilities, may not understand the shortened form of words.</dd> 
```